<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_img_exp</class>
 <widget class="QDialog" name="Dialog_img_exp">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>279</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Exportation images</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/icons/directoryExp.png</normaloff>:/icons/directoryExp.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 127);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>200</horstretch>
       <verstretch>200</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 127);</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Exportation setting</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_exp_icons">
       <property name="geometry">
        <rect>
         <x>218</x>
         <y>280</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Export Image</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>12</y>
         <width>521</width>
         <height>72</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select a Project</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_SITE">
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Site name</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>388</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="comboBox_p_name">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>.......</string>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>.......</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>529</width>
         <height>153</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Select a Site</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QComboBox" name="comboBox_location">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>.......</string>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>.......</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_divelog">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Divelog</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>448</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBox_artefact">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Artefact</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>408</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBox_pottery">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Pottery</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>438</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBox_anchor">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Anchor</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>438</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_open_dir">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>466</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Open folder IMAGE</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/icons/directoryExp.png</normaloff>:/icons/directoryExp.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
